hack/lib/test/junit.sh

```diff
@@ -183,19 +183,20 @@ function os::test::junit::generate_report() {
 #  export JUNIT_REPORT_NUM_FAILED
 function os::test::junit::internal::generate_report() {
     local report_type="$1"
-    os::util::ensure::system_binary_exists 'junitreport'
+    source .bingo/variables.env	# For JUNITREPORT
+    os::util::ensure::system_binary_exists $(JUNITREPORT)
```

Comment:
Bash syntax is _slightly_ different from Make:

```suggestion
    os::util::ensure::system_binary_exists ${JUNITREPORT}
```

This also applies to the two changes below.